docs/dev/shims.md

```diff
@@ -16,7 +16,7 @@ common code, maximize reuse, and minimize logic duplication.
 This is achieved by using a ServiceProvider pattern. All Shims implement the same API, 
 the suitable Shim implementation is loaded after detecting the current Spark build version
 attempting to instantiate our plugin. We use the
-[ShimLoader](../sql-plugin/src/main/scala/com/nvidia/spark/rapids/ShimLoader.scala)
+[ShimLoader](../../sql-plugin/src/main/scala/com/nvidia/spark/rapids/ShimLoader.scala)
```

Comment:
These pages are published to Github Pages, and this relative link will not work once published to that site.  We may want to point to this class on the `main` branch using an absolute link to the code, e.g.: https://github.com/NVIDIA/spark-rapids/blob/main/sql-plugin/src/main/scala/com/nvidia/spark/rapids/ShimLoader.scala

This applies to all relative links to code in this document.